frontend/src/scenes/notebooks/NotebookCommentButton/notebookCommentButtonLogic.ts

```diff
@@ -0,0 +1,32 @@
+import { events, kea, key, path, props } from 'kea'
+import { loaders } from 'kea-loaders'
+import { NotebookListItemType } from '~/types'
+
+import api from 'lib/api'
+
+import type { notebookCommentButtonLogicType } from './notebookCommentButtonLogicType'
+
+export interface NotebookCommentButtonProps {
+    sessionRecordingId: string
+}
+
+export const notebookCommentButtonLogic = kea<notebookCommentButtonLogicType>([
+    path((key) => ['scenes', 'session-recordings', 'NotebookCommentButton', 'multiNotebookCommentButtonLogic', key]),
+    props({} as NotebookCommentButtonProps),
+    key((props) => props.sessionRecordingId || 'no recording id yet'),
+    loaders(({ props }) => ({
+        notebooks: [
+            [] as NotebookListItemType[],
+            {
+                loadNotebooks: async () => {
+                    const response = await api.notebooks.list(props.sessionRecordingId)
+                    // TODO for simplicity we'll assume the results will fit into one page
+                    return response.results
+                },
+            },
+        ],
+    })),
+    events(({ actions }) => ({
+        afterMount: actions.loadNotebooks,
+    })),
```

Comment:
Not for now, but I think in the future this could make a lot of sense as a generic component / logic for `relatedNotebooks`. So that we could have it on recordings, feature flags, etc. But definitely for a follow up as this might become essentially the "Add to Notebook" button.